src/runtime/pkg/device/config/config.go

```diff
@@ -320,6 +327,9 @@ type VhostUserDeviceAttrs struct {
 	CacheSize uint32
 
 	QueueSize uint32
+
+	// Reconnect timeout for socket of vhost user block device
+	ReconnectTime int
```

Comment:
Do we want to expose this as a config in the configuration.toml?

Reply:
Yes, If we expose this as a config will be better